common/model-views.cpp

```diff
@@ -937,8 +939,24 @@ namespace rs2
                 }
             }
 
-            get_default_selection_index(res_values, default_resolution, &selection_index);
-            ui.selected_res_id = selection_index;
+            if (!strcmp(dev.get_info(RS2_CAMERA_INFO_PRODUCT_LINE), "D500") && 
+                !strcmp(s->get_info(RS2_CAMERA_INFO_NAME), "Stereo Module"))
+            {
+                ui.selected_res_map_used = true;
+                auto default_res = std::make_pair(1280, 960);
```

Comment:
We can take the default from the SDK no?
This way if we change in the future it will be updated automatically

Reply:
As discussed .We can add a new API in the future..